docs/Partitioner.md

```diff
@@ -0,0 +1,52 @@
+## Graph Partitioning Support in Glow
+
+### Overview
+Along with the increased size of neural networks, a large network may not be able to run on only one accelerator/device due to resource limitation (e.g. memory capacity). Therefore, it is necessary to divide up the network into several sub-networks and each of them can be executed on one accelerator. This problem can be abstracted as a Graph Partition problem [https://en.wikipedia.org/wiki/Graph_partition] : Given a neural network and n devices, we would like to split this network into minimal k sub-networks (where k <= n) , and for each sub-network, it can be run on one device with the device resource restriction.
+
+### Heterogeneous partition
+Glow now supports Heterogeneous Partition to address this problem. Given a network, and the information of a list of devices proviced by users, Glow will try to find a posssible partition or report an error.
```

Comment:
is this the main purpose of Heterogeneous partitioning? (or it might be useful if some ops are not supported on one device but supported on the other one)

Reply:
" it might be useful if some ops are not supported on one device but supported on the other one" -- This one has been addressed in later "Backend-Based-Partition".